pandas/tests/test_graphics_others.py

```diff
@@ -310,8 +311,8 @@ def _check_ax_limits(col, ax):
 
         # Two rows, one partial
         p = df.boxplot(['height', 'weight', 'age'], by='category')
-        height_ax, weight_ax, age_ax = p[0, 0], p[0, 1], p[1, 0]
-        dummy_ax = p[1, 1]
+        height_ax, weight_ax, age_ax = p[0], p[1], p[2]
```

Comment:
Apparently it is. I didn't quite realize that as I was getting the tests to pass :/